Source/WebCore/animation/TimelineRange.h

```diff
@@ -34,11 +34,24 @@ struct SingleTimelineRange {
 
     Name name { Name::Normal };
     Length offset;
+
+    bool operator==(const SingleTimelineRange& other) const = default;
```

Comment:
Don’t need the word "other" here, by the way. I wouldn’t hold up the pull request just for this, though.